Solutions/Corvus.SpecFlow.Extensions/Corvus/SpecFlow/Extensions/FunctionsController.cs

```diff
@@ -118,7 +119,7 @@ public sealed class FunctionsController
 
             lock (this.sync)
```

Comment:
I realise this hasn't changed, but this should probably be a lock with a timeout to stop the specs hanging for ever (until the specrunner times out) in the event of a failure or deadlock.

Fortunately @idg10 has a TimedLock which may even have made it into Corvus.

Reply:
A cursory search didn't reveal anything. I'm going to create a separate issue to address these concerns.

Reply:
And here it is: https://github.com/corvus-dotnet/Corvus.SpecFlow.Extensions/issues/79